web/book/src/compiler/README.md

```diff
@@ -0,0 +1,16 @@
+# Compiler
+
+The reference compiler implementation of PRQL is written in Rust. And there is a
+CLI for it.
+
+- [Reference implementation](./rust/README.md)
+  - [CLI](./rust/cli.md)
+
+## Other implementations
+
+### Go PRQL
```

Comment:
I'm very up for promoting things outside the official org. But is this still being developed? I see the most recent commit was almost a year ago. If it was a quick effort which is no longer being worked on, I'm not sure it makes sense to link to...